scripts/install2disk

```diff
@@ -114,6 +114,51 @@ EOF
         exit 0
         ;;
 
+    "pingan-ecs")
+        cat >cloud-config.yml<<EOF
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      METADATA_SERVER="100.68.11.8"
```

Comment:
It is better to pass in environment variables

Reply:
This is a fixed IP